File: examples/python/pdf_preview.py
import requests

API = "https://universal-pdf-screenshot-api.p.rapidapi.com"
headers = {
    "X-RapidAPI-Key": "YOUR_RAPIDAPI_KEY",
    "X-RapidAPI-Host": "universal-pdf-screenshot-api.p.rapidapi.com"
}

files = {"file": open("document.pdf", "rb")}
r = requests.post(f"{API}/pdf/preview", headers=headers, files=files)

open("preview.png", "wb").write(r.content)

File: examples/node/pdf_preview.js
const axios = require("axios");
const FormData = require("form-data");
const fs = require("fs");

const form = new FormData();
form.append("file", fs.createReadStream("document.pdf"));

axios.post(
  "https://universal-pdf-screenshot-api.p.rapidapi.com/pdf/preview",
  form,
  {
    responseType: "arraybuffer",
    headers: {
      ...form.getHeaders(),
      "X-RapidAPI-Key": "YOUR_RAPIDAPI_KEY",
      "X-RapidAPI-Host": "universal-pdf-screenshot-api.p.rapidapi.com",
    },
  }
).then(res => fs.writeFileSync("preview.png", res.data));
